src/service-override/extensionGallery.ts

```diff
@@ -22,13 +22,11 @@ import { RemoteExtensionsScannerService } from 'vs/workbench/services/remote/com
 import { ExtensionRecommendationNotificationService } from 'vs/workbench/contrib/extensions/browser/extensionRecommendationNotificationService'
 import { ExtensionTipsService } from 'vs/platform/extensionManagement/common/extensionTipsService'
 import { ExtensionManagementService } from 'vs/workbench/services/extensionManagement/common/extensionManagementService'
-import 'vs/workbench/contrib/extensions/browser/extensions.contribution'
-import 'vs/workbench/contrib/extensions/browser/extensions.web.contribution'
-import 'vs/workbench/contrib/logs/common/logs.contribution'
 import { IRemoteUserDataProfilesService, RemoteUserDataProfilesService } from 'vs/workbench/services/userDataProfile/common/remoteUserDataProfiles'
 import { ExtensionEnablementService } from 'vs/workbench/services/extensionManagement/browser/extensionEnablementService'
-import 'vs/workbench/services/extensionManagement/browser/extensionBisect'
```

Comment:
Last time I noticed we need the bisect, but I assume that it's now contributed in another way?

Reply:
Oh it seems it'seven imported by missing-services.ts, it may be an issue

Reply:
Ok the risk seems to only be to include some actions and make the bundle bigger for nothing if the extension gallery service override is not used